### ISSUE TITLE
System Monitor: Installing ANY of the "NEw Pages" is broken, installing two in a row shows even the select mechnism within System Monitor is broken, having two pages selected at once

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.9#252]

**Comment by Pointedstick** [comment]
Can confirm the newly-installed pages being broken; they don't show any information in their sensors. Restarting the app does not fix it.

Cannot confirm having two pages selected at once though. How did you make that happen?

**Comment by Sodivad** [comment]
I've installed every available page from the store. Some work, some don't, some work partially. It depends on what sensors they use:

- **CPU** sensors work
- **GPU** Sensors work
- **Memory** works
- **lmsensors** do not work because they are highly system specific
- **Disk** sensors do not work because we use the  uuid to identify them
- **Network** do not work generally since the link name (i.e. wlp3s0) identifies the device
- **Batteries** use an id based on their serial which is also not portable
- Every **All Foo**  sensor because they just sum every up

In conclusion it depends on if the sensor id contains non-portable device identifier (for example partition uuid) or portable ones like 'gpu1', 'cpu2' or 'swap'

**Comment by Pointedstick** [comment]
Maybe there's a way the software can warn people uploading content that their pages include non-portable sensor IDs? Seems like the ability to do that is an undocumented foot-gun. Or maybe we can prohibit it entirely.
